src/components/news-article.js

```diff
@@ -1,17 +1,34 @@
 import React from 'react'
 import SbEditable from 'storyblok-react'
 import Components from './components.js';
+import { StoryblockFormatter } from '../utils/RichTextFormatter';
+import "./layout.css";
+import { Helmet } from "react-helmet";
 
-const StoryblokClient = require('storyblok-js-client')
 
-const Feature = (props) => (
-  <SbEditable content={props.blok}>
-    <div className="col-8">
-        <h1>{props.blok.title}</h1>
-        <div>{props.blok.caption}</div>
-        <div>{JSON.stringify(props.blok.content)}</div>
-    </div>
-  </SbEditable>
-)
+const Feature = ({ blok, children }) => {
+	
+	return (
+		<SbEditable content={blok}>
+		<Helmet>
```

Comment:
Installed an extension for OG unfortunately it doesn't pick it up.. not sure why as I can see the image in the inspector when I check the <head> tag